### ISSUE TITLE
Changing directory in the build command doesn't work

### ISSUE THREAD

**Issue by bousquetm** [slynch8/10x#327]
My build process is a .bat file that I launch with the following argument: `$(WorkspaceDirectory)\test.bat` (`$(WorkspaceDirectory)` is set to `C:\dev\Test`)

The test.bat contains only 3 lines:
```
cd
cd ..
cd
```

And the result is:
```
----- Build Release x64 -----
Executing build command:
C:/dev/Test\test.bat

C:\Program Files\PureDevSoftware\10x>cd
C:\Program Files\PureDevSoftware\10x

C:\Program Files\PureDevSoftware\10x>cd .. 

C:\Program Files\PureDevSoftware>cd
C:\Program Files\PureDevSoftware
===== Success : 0 errors, 0 warnings =====
```

Which shows that the directory change is not taken into account at all. Also maybe it would make more sense if the startup directory were `$(WorkspaceDirectory)` instead of the install dir of 10x.

I'm using Windows 10 with 10x version 0.27.17.0.

**Comment by slynch8** [comment]
Fixed in (0.27.19)

Not sure why the cd .. wasn't working, but seems to work now.

**Comment by bousquetm** [comment]
Confirmed fixed in 0.27.19.
